src/ripple/rpc/handlers/LedgerEntry.cpp

```diff
@@ -51,7 +51,15 @@ doLedgerEntry(RPC::JsonContext& context)
 
     if (context.params.isMember(jss::index))
     {
-        if (!uNodeIndex.parseHex(context.params[jss::index].asString()))
+        // Document states "index" field is a string. check for this apiVersion
```

Comment:
What document are you referring to ? xrpl.org ? You can make this clear in the comments and remove ambiguity.